lava/lava-job-definitions/shared/templates/component-update-job.yaml

```diff
@@ -0,0 +1,34 @@
+{% extends "shared/templates/base.yaml" %}
+
+{% set job_name = component_name + " image update via " + update_method %}
+{% set lxc_creation = true %}
+{% set lxc_name = component_name + "-image-update-" + update_method + "-lxc" %}
+{% set payloads_url = pipeline_data["payloads_url"] %}
+
+{% block testplan %}
+- test:
+    timeout:
+      minutes: 30
+    namespace: lxc
+    definitions:
+
+    {{ macros.avahi_discovery() | indent }}
```

Comment:
ahavi_discovery must come after  create_python_environment because it is a pytest